ios/Classes/SwiftAirshipPlugin.swift

```diff
@@ -452,10 +452,7 @@ public class SwiftAirshipPlugin: NSObject, FlutterPlugin {
             return try AirshipJSON.wrap(flag).unWrap()
 
         case "featureFlagManager#trackInteraction":
-            guard let arg = try? call.requireAnyArg() as? String else {
-                throw AirshipErrors.error("Call requires a String argument")
-            }
-
+            let arg = try call.requireStringArg()
```

Comment:
I also removed the optional try, I think that's the reason why it was failing before